tdrs-backend/tdpservice/parsers/util.py

```diff
@@ -291,3 +133,17 @@ def transform_to_months(quarter):
 def month_to_int(month):
     """Return the integer value of a month."""
     return datetime.strptime(month, '%b').strftime('%m')
+
+def get_rpt_month_year_list(year, quarter):
```

Comment:
I'm seeing some redundancy with this function `get_rpt_month_year_list()`, `transform_to_months()`, and `month_to_int()`. While it may be an unnecessary optimization, my brain feels weird about this recurring section of code especially in regards to `transform_to_months()`.

Reply:
@andrew-jameson, let me know what you think about the change I made with respect to this